### PR TITLE
Use yarn, set up css grid

### PR DESCRIPTION
**Pull request by sravanti**
A few things in this PR: 
- Remove package-lock.json -- we should only be using yarn for package management. 
- Introduce CSS grid -- visually this still looks rough, but the pieces are in place to follow the mockup Samantha made.

Mockup:
Open Transit Wireframes

Current status:
Screen Shot 2019-03-20 at 8 42 53 PM

I also did some code cleanup, removing unnecessary divs and using Emotion for our CSS-in-JS lib. I also moved the big address object into its own constants file.

@hathix lmk who from front-end I should add to this review for a heads up. I'll also merge latest master to avoid merge conflicts.

### CODE REVIEW

File: frontend/yarn.lock [hunk 1]
```diff
@@ -789,31 +884,93 @@
...
 "@date-io/date-fns@^1.1.0":
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/@date-io/date-fns/-/date-fns-1.1.0.tgz#91cd7703042513a6a0056a344b25cd74a4df643e"
+  integrity sha512-FMRhYWfoGiIXdN4xWAArpkdEbqsg2Fr+6Yda7Np2eVWCNx6gSMYsHIM51IIcI+3762ajYbhoEYjHYXVFNZIk1g==
   dependencies:
     "@date-io/core" "^1.1.0"
 
+"@emotion/cache@^10.0.9":
+  version "10.0.9"
+  resolved "https://unpm.uberinternal.com/@emotion%2fcache/-/cache-10.0.9.tgz#e0c7b7a289f7530edcfad4dcf3858bd2e5700a6f"
```
Comment (youngj) [inline frontend/yarn.lock]: This tries to install some packages from Uber's corporate network, which throws a "401 Unauthorized" error for non-Uber employees :)

**Comment by hathix** [comment]
Try getting Chris or Matt's feedback too - I added them as reviewers.

**Comment by hathix** [comment]
At any rate, I like adding the css grid. Haven't looked into emotion but if others like it, it's good with me

File: frontend/package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "@date-io/date-fns": "^1.1.0",
     "axios": "^0.18.0",
     "bootstrap": "^4.3.1",
+    "emotion": "^10.0.9",
```
Comment (ckrajewski) [inline frontend/package.json]: @sravanti 
I got this error when doing yarn install: (I got a 401 unauthorized)
error An unexpected error occurred: "https://unpm.uberinternal.com/create-emotion/-/create-emotion-10.0.9.tgz: Request failed \"401 Unauthorized\"".
npm install worked though...

**Comment by sravanti** [comment]
@youngj and @ckrajewski thank you so much for pointing that out! i have no idea how the uber package manager got in there, but great catch :) i'll update later today!

**Comment by sravanti** [comment]
As a FYI I'm going to remove yarn and only use NPM for this project since, as Neil mentioned, there's been some debate. Sound good?

**Comment by hathix** [comment]
Yup, as long as we're consistent I don't really care whether we use Yarn or NPM.

**Comment by hathix** [comment]
So the only thing left is to remove the Uber package, right?

**Pull request merged**